login-workflow/example3/src/screens/ExampleHome.tsx

```diff
@@ -89,20 +87,26 @@ export const ExampleHome: React.FC<React.PropsWithChildren> = () => {
                         <Spacer />
                         <UserMenu
                             onClick={(): void => setOpen(!open)}
-                            avatar={<Avatar>MH</Avatar>}
-                            menu={
-                                <Menu open={open} onClose={(): void => setOpen(false)}>
-                                    <MenuItem onClick={(): void => setOpen(false)}>{`${t('Blui:MYACCOUNT')}`}</MenuItem>
-                                    <MenuItem
-                                        onClick={(): void => {
-                                            setOpen(false);
-                                            logOut();
-                                        }}
-                                    >
-                                        {`${t('Blui:LOGOUT')}`}
-                                    </MenuItem>
-                                </Menu>
-                            }
+                            avatar={<Avatar>AV</Avatar>}
+                            menuGroups={[
+                                {
+                                    items: [
+                                        {
```

Comment:
Add the icons to these menuItems

Reply:
Consider adding a language dropdown menu too next to the user menu (not necessary, but if it's quick and easy, go for it)